inventory.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 Mender Software AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package main
+
+type InventoryAttribute struct {
+	Name  string      `json:"name"`
+	Value interface{} `json:"value"`
```

Comment:
I see this now. I thought JSON marshaler is bit smarter and detects that there is a single string inside slice. From the other hand this would get things even worse. Anyway, let's stick to what was agreed.